source/includes/gcp/_list_firewall_rules.md

```diff
@@ -0,0 +1,156 @@
+               ]
+            },
+            {
+               "IPProtocol":"udp",
+               "ports":[
+                  "0-65535"
+               ]
+            },
+            {
+               "IPProtocol":"icmp"
+            }
+         ],
+         "creationTimestamp":"2019-06-05T13:08:23.802-07:00",
+         "description":"Allow internal traffic on the default network",
+         "direction":"INGRESS",
+         "disabled":false,
+         "id":"95649327595495384",
+         "kind":"compute#firewall",
+         "logConfig":{
+            "enable":false
+         },
+         "name":"default-allow-internal",
+         "network":"https://www.googleapis.com/compute/v1/projects/testing-uxx/global/networks/default",
+         "priority":65534,
+         "selfLink":"https://www.googleapis.com/compute/v1/projects/testing-uxx/global/firewalls/default-allow-internal",
+         "sourceRanges":[
+            "10.128.0.0/9"
+         ],
+         "action":"allow"
+      },
+      {
+         "allowed":[
+            {
+               "IPProtocol":"tcp",
+               "ports":[
+                  "3389"
+               ]
+            }
+         ],
+         "creationTimestamp":"2019-06-05T13:08:23.825-07:00",
+         "description":"Allow RDP from anywhere",
+         "direction":"INGRESS",
+         "disabled":false,
+         "id":"8521921759829925848",
+         "kind":"compute#firewall",
+         "logConfig":{
+            "enable":false
+         },
+         "name":"default-allow-rdp",
+         "network":"https://www.googleapis.com/compute/v1/projects/testing-uxx/global/networks/default",
+         "priority":65534,
+         "selfLink":"https://www.googleapis.com/compute/v1/projects/testing-uxx/global/firewalls/default-allow-rdp",
+         "sourceRanges":[
+            "0.0.0.0/0"
+         ],
+         "action":"allow"
+      },
+      {
+         "allowed":[
+            {
+               "IPProtocol":"tcp",
+               "ports":[
+                  "22"
+               ]
+            }
+         ],
+         "creationTimestamp":"2019-06-05T13:08:23.814-07:00",
+         "description":"Allow SSH from anywhere",
+         "direction":"INGRESS",
+         "disabled":false,
+         "id":"4638036624227540952",
+         "kind":"compute#firewall",
+         "logConfig":{
+            "enable":false
+         },
+         "name":"default-allow-ssh",
+         "network":"https://www.googleapis.com/compute/v1/projects/testing-uxx/global/networks/default",
+         "priority":65534,
+         "selfLink":"https://www.googleapis.com/compute/v1/projects/testing-uxx/global/firewalls/default-allow-ssh",
+         "sourceRanges":[
+            "0.0.0.0/0"
+         ],
+         "action":"allow"
+      }
+   ],
+   "metadata":{
+      "recordCount":4
+   }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/instances</code>
```

Comment:
replace 'instances' for 'firewallrules' so the GET line represents the call to make to the API.